src/main/resources/META-INF/resources/admin.xhtml

```diff
@@ -28,43 +28,64 @@
         </p:dialog>
 
         <div class="wrapper">
-            <div id="loader" class="load-bar" style="display: none">
-                <div class="bar"></div>
-                <div class="bar"></div>
-                <div class="bar"></div>
-            </div>
-            <header class="main-header">
-                <a href="#{request.contextPath}/" class="logo">
-                    <span class="logo-mini">
-                        <ui:insert name="logo-mini"/>
-                    </span>
-                    <span class="logo-lg">
-                        <ui:insert name="logo-lg"/>
-                    </span>
-                    <span class="logo-lg">
-                        <ui:insert name="logo"/>
-                    </span>
-                </a>
-                <nav class="navbar navbar-static-top" role="navigation">
-                    <a href="#" class="sidebar-toggle" data-toggle="offcanvas" role="button">
-                        <span class="sr-only"></span>
-                    </a>
-                    <div class="navbar-custom-menu">
-                        <ui:insert name="top-menu"/>
-                    </div>
-                </nav>
-            </header>
-
-            <!-- side menu -->
-            <aside id="sidebar" class="main-sidebar slideout-menu">
-                <div>
-                    <ui:insert name="menu-begin"/>
-                    <section class="sidebar">
-                        <ui:insert name="menu"/>
-                    </section>
-                    <ui:insert name="menu-end"/>
-                </div>
-            </aside>
+        	<ui:fragment rendered="#{empty renderLoader or renderLoader eq 'true'}">
```

Comment:
Can't we reuse #{adminConfig.renderAjaxStatus} instead of creating a new variable here?

Also note that we can have a js error if the loader is not rendered, because ajaxStatus will try to display it, [see here](https://github.com/adminfaces/admin-template/blob/fdd781aefbb98acf2d9c4adc6db05cb873e9f09a/src/main/resources/META-INF/resources/admin.xhtml#L20) and [here](https://github.com/adminfaces/admin-template/blob/fdd781aefbb98acf2d9c4adc6db05cb873e9f09a/src/main/resources/META-INF/resources/js/admintemplate.js#L204)